build.sbt

```diff
@@ -286,6 +286,7 @@ lazy val dist = {
         (crossTarget in generic).value / "genericModel.jar" -> "model/genericModel.jar",
         (crossTarget in demo).value / s"demoModel.jar" -> "model/demoModel.jar",
         (crossTarget in flinkProcessManager).value / s"nussknacker-flink-manager.jar" -> "managers/nussknacker-flink-manager.jar",
+        (crossTarget in flinkPeriodicProcessManager).value / s"nussknacker-flink-periodic-manager.jar" -> "managers/nussknacker-flink-periodic-manager.jar",
```

Comment:
I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)